### ISSUE TITLE
Thursday

### ISSUE THREAD

**Issue by Codingmaestre2015** [WestMEC-Coding-Session1/Week-28-Bell-Work#3]
**Please respond to the following question:**

Does PHP code embedded in an HTML file have to be on its own line, or can I embed it in an HTML line, like as part of an HTML tag attribute?

Explain

**Comment by osborn-daucen-wm** [comment]
First!

**Comment by schweikert-zachary-wm** [comment]
PHP tags can be anywhere within an HTML file. They can do various things by placing them in various places.

**Comment by thorpe-sean-wm** [comment]
PHP can be written anywhere as long a you keep it inside a <?php ... ?> tag.

**Comment by le-ben-wm** [comment]
PHP can be embedded almost anywhere as long as there is an opening tag and closing tag around the code.

**Comment by brey-matthew-wm** [comment]
It could be in an html line, or just at the bottom, you just have to use the <? tag to open the php, and the page has to be .php not .html

**Comment by mason-gavin-wm** [comment]
PHP can be almost anywhere just as long as you put it in with <?php ... ?>.

**Comment by gonzalez-jose-wm** [comment]
PHP can be almost anywhere as long as it is inclosed between <?php ... ?> .

**Comment by clocksin-nick-wm** [comment]
It can be anywhere in an html form as long as its in a php tag in a php type html file.

**Comment by breeah-wm** [comment]
PHP can be written in HTML if written with <?php ...?>.

**Comment by riggs-jacob-wm** [comment]
It is possible to have your PHP code anywhere as long it is inside <?php  ... ?> tag.

**Comment by Keyan-Ngai-wm** [comment]
Php can be just about anywhere in the html as long as you use <?php...?>.

**Comment by burton-jacob-wm** [comment]
It can be embedded inside a html line,tag, and placed almost anywhere in a php file.

**Comment by Garcia-Rogelio-WM** [comment]
It can be anywhere it can be embedded in between and outside the html

**Comment by mcclure-calvin-wm** [comment]
PHP can be anywhere in html as long as it is enclosed in tags.

**Comment by cunningham-corbin-wm** [comment]
You could if you wanted never press the "enter" key and your code would be all on one line, so technically PHP could be on the same line as HTML, but it is better practice to separate out your code by pressing the "enter" key. You do have to use the <?php ?> tag.

**Comment by deherrera-marques-wm** [comment]
You can do it in html as long as you do <?php ?> tag, and if you do that it can be anywhere in an php file.

**Comment by casey-leevon-wm** [comment]
<h1>PHP can be put anywhere in the file.</h1>

**Comment by diaz-andy-wm** [comment]
Php can be embedded within html but as its own tag. It can be separated from other text with html tags for css reasons but no it does not have to be between the html code.

**Comment by hogencamp-devon-wm** [comment]
Yes...You can put PHP anywhere and it will work. If you put PHP tags inside of HTML tags you can have a dynamic output that is generated.

**Comment by steele-ariel-wm** [comment]
PHP can be written anywhere in the HTML document; however, you have to open the PHP tag with <?php

**Comment by pierce-eric-wm** [comment]
Php can be anywhere in the file(php) so it does not have to be in a certain place all of the time and php is written by <?php ..... ?>

**Comment by marquez-mollee-wm** [comment]
PHP can be put anywhere that the PHP tag can be placed which is basically anywhere in the HTML.

**Comment by athina-bitsui-wm** [comment]
the php can go any where in the html file, but as long it has the <?php..?> with it.

**Comment by shephard-azaniyah-wm** [comment]
Php can be in the same file as html as well as being put anywhere in it. In order for it to work, a specific tag must be implemented in order for it to function correctly.
